orttraining/orttraining/core/optimizer/megatron_transformer.cc

```diff
@@ -136,7 +137,6 @@ bool MegatronTransformer::PartitionWeightByColumn(const Graph& graph, const Node
     LOGS_DEFAULT(WARNING) << "Initializer tensor's rank is " << rank << " (expected to be 1 or 2).";
     return false;
   }
-
```

Comment:
are the blank line removal by intention?

Reply:
reverting